docs/tutorials/auto_prepare_commit_message.md

```diff
@@ -0,0 +1,27 @@
+# Automatically prepare message before commit
+
+## About
+
+To automatically prepare a commit message prior to committing, you can use a [Git hook](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks).
```

Comment:
I'm not familiar with this hook, could you add a bit more about the motivation, and explaining the workflow for this?